AUTHORS

```diff
@@ -89,6 +89,7 @@ Also thanks to:
     * Lesueur Benjamin (Valkirie)
     * Maarten Meuris (Nyerguds)
     * Mark Olson (markolson)
+	* Markus Hartung (hartmark)
```

Comment:
This is a reverse case of our code, the other entries use spaces instead of a tab.